app/Services/Media.php

```diff
@@ -229,12 +229,18 @@ public function getInfo(SplFileInfo $file)
             return;
         }
 
-        $track = array_get($info, 'comments.track_number', [0])[0];
+        if (!$track = array_get($info, 'comments.track', [0])[0]) {
```

Comment:
Would you rather just a bunch of `elseif`s? Otherwise I'm not sure of a better way to account for multiple ways of getting the same data.